# HG changeset patch
# User Andrew Cooper <andrew.cooper3@citrix.com>
# Date 1394707855 0
# Node ID 86b84e26a03e91e479601f34e3aea8fff2a3abd1
# Parent  bb44f16a40321fea2949747c593ef223f90c088d
tools/xen-mceinj: Fix depency for the install rule

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
Acked-by: Liu Jinsong <jinsong.liu@intel.com>
Acked-by: Ian Campbell <Ian.Campbell@citrix.com>
CC: Ian Jackson <Ian.Jackson@eu.citrix.com>

diff -r bb44f16a4032 -r 86b84e26a03e tools/tests/mce-test/tools/Makefile
--- a/tools/tests/mce-test/tools/Makefile
+++ b/tools/tests/mce-test/tools/Makefile
@@ -10,7 +10,7 @@ CFLAGS += $(CFLAGS_xeninclude)
 .PHONY: all
 all: xen-mceinj
 
-install: 
+install: xen-mceinj
 	$(INSTALL_PROG) xen-mceinj $(DESTDIR)$(SBINDIR)
 
 .PHONY: clean
